gradle.properties

```diff
@@ -0,0 +1,3 @@
+cdvCompileSdkVersion=android-21
```

Comment:
These properties are required by ```Cordova``` library project, to add as a ```Gradle``` dependency. Unfortunately, it doesn't work with ```Marshmallow``` quite yet - ```targetApi``` I mean.